### DISCUSSION TITLE
Importing Patches?

### DISCUSSION THREAD

**Discussion by garygeeg** [DISTRHO/Cardinal#259] [Q&A] [answered]
Maybe a silly question but how do you load the patches available here? I assumed you imported them with the File menu but that gives me an error..? This on a Windows PC in Reaper. Thanks.

![cardinal_import_error](https://user-images.githubusercontent.com/95886416/174289278-3780e1e2-1e1d-47cd-9bf8-d5e83fab579d.png)

**Comment by falkTX** [comment, marked as answer]
Available "here" where? If you are trying to download from github, make sure to download the raw file instead of the html page.

**Reply by garygeeg** [reply]
Spot on, thanks. 
In the meantime I DLed a couple from Patch Arena which just worked, obviously a different download mechanism.

Thanks again.

**Reply by dromer** [reply]
What is "Patch Arena"? Do you mean Patchstorage perhaps?

**Reply by garygeeg** [reply]
Sorry, yes, I do. I was looking at the old PatchArena site archived on Wayback Machine the other day and had that on my mind.
